linux_os/guide/system/accounts/accounts-pam/display_login_attempts/rule.yml

```diff
@@ -57,20 +54,20 @@ references:
 
 platform: package[pam]
 
-ocil_clause: '"pam_lastlog" is missing from "{{{ pam_lastlog_path }}}" file, or the silent option is present'
+ocil_clause: '"pam_lastlog.so" is not properly configured in "{{{ pam_lastlog_path }}}" file'
 
 ocil: |-
     Verify users are provided with feedback on when account accesses last occurred with the following command:
 
     <pre>$ sudo grep pam_lastlog {{{ pam_lastlog_path }}}
 
-    session required pam_lastlog.so showfailed</pre>
+    session [default=1] pam_lastlog.so nowtmp showfailed</pre>
```

Comment:
for now the Ubuntu 20.04 STIG benchmark still mentions:
`session required pam_lastlog.so showfailed`

Reply:
Fixed